src/main/java/com/stripe/param/issuing/CardCreateParams.java

```diff
@@ -3392,7 +3389,7 @@ public Builder setType(Type type) {
       }
 
       /** Packaging options. */
-      public Builder setType(EmptyParam type) {
+      public Builder setType(String type) {
```

Comment:
What's this about?

Reply:
That one I'm not sure, I think it's because they changed the way string/enums work in their API. cc @ob-stripe

Reply:
Internally, the `type` parameter was changed from a `StringParam` with a set of `values` to our more modern `APIEnumParam ` abstraction with `allow_all_strings_in_schema` set to `true` to avoid a breaking change.

In the OpenAPI spec, this caused `type` to be updated from a `string` with two possible `enum` values to a polymorphic `anyOf` type: either a `string` with two possible `enum` values (i.e. the previous definition), or a simple `string` (due to `allow_all_strings_in_schema: true`).

So it's expected that we now have two overloads for `type` (one with the `Type` enum type and one with `String`). I can't figure out why we had an overload with `EmptyParam` before though, since that was not a valid value for this parameter.

This is technically a breaking change, but I think it's fine to release this as a minor since I'm fairly sure using `EmptyParam` would have caused the API request to fail anyway.